### DISCUSSION TITLE
Print Orders API interaction?

### DISCUSSION THREAD

**Discussion by tophattwaffle** [etsy/open-api#1179] [Q&A]
I'm looking for a way to get the PDF files that are generated from this dialogue:
![image](https://github.com/etsy/open-api/assets/6774125/d3556736-89ed-4d04-b580-02f85544ac9c)

I want to write a feature for my project that will automatically print the packing slip for new orders. I specifically want the PDF files generated by Etsy as I like how they look compared to fully designing my own from the information returned by the getShopReceipt endpoint.

**Comment by Gareth064** [comment]
You can't interact with the shipping label printing feature via the API unfortunately. That can only be done via the Web App.

**Comment by turboturtleD** [comment]
No endpoint for the packing slip and shipping label pdfs.  IMO though, the design work is the hard part, so just mimic the Etsy template with some HTML and print that instead.
